drivers/idle/ndl/ndl.c

```diff
@@ -118,7 +118,7 @@ static int ndl_do_init(void)
 	}
```

Comment:
wult: drivers: ndl: Map MMIO with pci_ioremap_bar()

Cherry-picked this patch.